src/test/.eslintrc.js

```diff
@@ -16,12 +16,23 @@ module.exports = {
 
     // Adding more errors now
     'testing-library/no-manual-cleanup': 'error',
-    'testing-library/no-wait-for-empty-callback': 'error',
     'testing-library/no-wait-for-snapshot': 'error',
     'testing-library/prefer-explicit-assert': 'error',
     'testing-library/prefer-presence-queries': 'error',
     'testing-library/prefer-wait-for': 'error',
 
+    // Disable some rules that are in the "recommended" part.
+    // This is a purely stylistic rule
+    'testing-library/render-result-naming-convention': 'off',
+    // This disallows using `container`, but this is still useful for us sometimes
+    'testing-library/no-container': 'off',
+    // This disallows using direct Node properties (eg: firstChild), but we have
+    // legitimate uses:
+    'testing-library/no-node-access': 'off',
+    // Disable until https://github.com/testing-library/eslint-plugin-testing-library/issues/359
+    // is fixed.
+    'testing-library/await-async-query': 'off',
```

Comment:
I only kept the rule [prefer-screen-queries](https://github.com/testing-library/eslint-plugin-testing-library/blob/main/docs/rules/prefer-screen-queries.md) because I think it's important and didn't need too much work to fix existing cases.